packages/cozy-client/src/ClientErrors.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback, useState, useEffect } from 'react'
+import { useClient } from 'cozy-client'
+import QuotaAlert from 'cozy-ui/react/QuotaAlert'
+import { pull } from 'lodash'
+
+/**
+ * Rendering functions for client fetch errors by status code
+ *
+ * @private
+ * @type {object}
+ */
+const byHttpStatus = {
+  413: QuotaError
+}
```

Comment:
I'm wondering if there is a way to make that pass the "grep test Component". I used to search for `<ComponentName` and not only `ComponentName` to find where the component is used.

Reply:
Is the comment ok for you ?